src/Evolution/Systems/ScalarTensor/BoundaryConditions/Factory.hpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Domain/BoundaryConditions/Periodic.hpp"
+#include "Evolution/Systems/CurvedScalarWave/BoundaryConditions/AnalyticConstant.hpp"
+#include "Evolution/Systems/CurvedScalarWave/BoundaryConditions/DemandOutgoingCharSpeeds.hpp"
+#include "Evolution/Systems/CurvedScalarWave/BoundaryConditions/Factory.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BoundaryCondition.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/DemandOutgoingCharSpeeds.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/DirichletAnalytic.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/DirichletMinkowski.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Factory.hpp"
+#include "Evolution/Systems/ScalarTensor/BoundaryConditions/BoundaryCondition.hpp"
+#include "Evolution/Systems/ScalarTensor/BoundaryConditions/ProductOfConditions.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ScalarTensor::BoundaryConditions {
+
+static constexpr size_t Dim = 3;
```

Comment:
We don't want global variables like this. Either template the individual type aliases on `Dim` (possibly with a default) or just hard-code the 3 in. Global variables are the root of all evil, constexpr ones are slightly less evil, but not by much.

Reply:
I've hardcoded to just 3